osf/models/provider.py

```diff
@@ -198,6 +199,9 @@ def absolute_api_v2_url(self):
 
 
 class RegistrationProvider(AbstractProvider):
+    REVIEWABLE_RELATION_NAME = 'registrations'
```

Comment:
drive by: what would the consequence of just making this 'draft_registrations' be? That would eliminate the need for the MODERATION_MACHINE_STATE_RELATION, correct?

Reply:
Am I correct in understanding that this is helping us annotate the registration itself with the state?

Reply:
1. Yep! good catch
2. Yep, the only real relevancy of that was in `get_reviewable_state_counts` but as you point that function can just as easily grab a DraftRegistration as a Registration, it just didn't occur to me initially because the serializer is serializing registrations.